apps/src/lib/ui/accounts/ChangeEmailController.js

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import ReactDOM from "react-dom";
+import color from '../../../util/color';
+import ChangeEmailModal from './ChangeEmailModal';
+import {hashEmail} from '../../../code-studio/hashEmail';
+
+export const ENCRYPTED_EMAIL_PLACEHOLDER = '***encrypted***';
```

Comment:
Should this be loc'ed?

Reply:
Probably, but that's an existing issue...